hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/multipart/S3ExpiredMultipartUploadsAbortRequest.java

```diff
@@ -197,9 +198,11 @@ private void updateTableCache(OzoneManager ozoneManager,
     OMMetadataManager omMetadataManager = ozoneManager.getMetadataManager();
     OmBucketInfo omBucketInfo = null;
     BucketLayout bucketLayout = null;
+    OMLockDetails omLockDetails = null;
     try {
-      acquiredLock = omMetadataManager.getLock()
+      omLockDetails = omMetadataManager.getLock()
           .acquireWriteLock(BUCKET_LOCK, volumeName, bucketName);
+      acquiredLock = omLockDetails.isLockAcquired();
```

Comment:
This omLockDetails is not set to the final OMClientResponse.

Reply:
Updated.